jelly/src/main/java/org/kohsuke/stapler/jelly/HTMLWriterOutput.java

```diff
@@ -64,18 +64,11 @@ private HTMLWriterOutput(HTMLWriter hw, OutputFormat fmt, boolean escapeText) {
         hw.setEscapeText(escapeText);
         this.htmlWriter = hw;
         this.format = fmt;
+        format.setExpandEmptyElements(true);
     }
 
     @Override public void close() throws IOException {
         htmlWriter.close();
     }
 
-    /**
-     * False to turn off HTML handling and reenable {@code />} for any empty XML element.
-     * True to switch back to default mode with HTML handling.
-     */
-    public void useHTML(boolean enabled) {
```

Comment:
Can be reverted to avoid the major release

Reply:
Maybe you can set it as @Deprecated and print something in logs to make it clear that this method is not going to be taking into account when rendering xml.

Reply:
Great idea. It was very useful to remove it as I was building and searching for any uses so there would be a hard failure. Now that's done, it's good to put it back into prevent anything out there that I may not have caught.